starter_code/src/movies.js

```diff
@@ -76,6 +81,18 @@ function ratesAverage(arr) {
 }
 
 // Iteration 5: Drama movies - Get the average of Drama Movies
+function dramaMoviesRate(arr) {
+  const arrDram = arr.filter(e => {
+    return e.genre.includes("Drama");
+  });
+
```

Comment:
To check if there is no Drama movie (one of the tests):
if(!arrDram.length) {
  return 0;
}